README.md

```diff
@@ -1,72 +1,70 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Todo list App
 
-> Description the project.
 
+> This branch (todo__list) We did it by contributing with my coding partner we did in this branch the followings 👌 :
+- [x] Set up a new project with webpack.
+- [x] Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder.
+- [x] Create The index.html file must be set as a template using the HTML Webpack Plugin.
+- [x] Create an index.js file and set an array of some simple to do tasks (array of objects).
+- [x]  Each task object should contain three keys
+- [x] Write a function to iterate over the tasks array and populate an HTML list item element for each task.💯
+- [x] On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
+- [x] Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader.
```

Comment:
- [ ] Kindly update this section. This is supposed to be the general description of the project and not the pull request's description 👍